src/main/scala/zio/prelude/AssociativeCompose.scala

```diff
@@ -14,9 +14,12 @@ trait AssociativeCompose[:=>[-_, +_]] {
     eq.equal(ad1, ad2)
   }
 }
-object AssociativeCompose             {
-  implicit val FunctionAssociativeCompose: AssociativeCompose[Function] =
-    new AssociativeCompose[Function] {
+
+object AssociativeCompose {
+  implicit val FunctionIdentityCompose: IdentityCompose[Function] =
+    new IdentityCompose[Function] {
+      def identity[A]: A => A = (a: A) => a
+
```

Comment:
WDYT about removing instance in here and restore the one in IdentityCompose ?

Reply:
I have study others PR related to this. Now I understand why that way and not the other :)